src/term/mod.rs

```diff
@@ -291,7 +292,7 @@ impl Term {
         val.to_string(def_names),
         nxt.to_string(def_names)
       ),
-      Term::Sup { fst, snd } => format!("{{{} {}}}", fst.to_string(def_names), snd.to_string(def_names)),
+      Term::Sup { fst, snd, .. } => format!("{{{} {}}}", fst.to_string(def_names), snd.to_string(def_names)),
```

Comment:
We should probably readback and print the label of sups, since they can appear in the result (in non-lambda programs)